lib/bolt/cli.rb

```diff
@@ -504,6 +508,14 @@ def generate_types
       0
     end
 
+    def initialize_project
+      path = File.expand_path(options[:object] || Dir.pwd)
+      FileUtils.mkdir_p(path)
+      ok = FileUtils.touch(File.join(path, 'bolt.yaml'))
```

Comment:
I think we typically only save this to a var if we want to pass `ok` to the outputter. Since we don't need to do that here this can just be `FileUtils.touch(File.join(path, 'bolt.yaml')) ? 0 : 1`.